README.md

```diff
@@ -54,7 +54,12 @@
 ### Key Features <a name="key-features"></a>
 
```

Comment:
- [ ] In the presentation video, Please make sure also to show some code that you want to highlight as requested in the project requirements. Currently, I can't see any highlighted code in the video.

![image](https://github.com/martinkarugaba/budget_app/assets/83803414/63fb4027-be60-49a9-8b63-152a06f1ec9a)